src/resolution.rs

```diff
@@ -48,12 +48,14 @@ impl Resolve {
 pub struct ResolvePackageMetadata {
     pub realm: Realm,
     pub server_only: bool,
+    pub source_registry: PackageSourceId,
 }
 
 pub fn resolve(
     root_manifest: &Manifest,
     try_to_use: &BTreeSet<PackageId>,
     package_sources: &PackageSourceMap,
+    source_registry: &PackageSourceId,
```

Comment:
Do we ever pass anything other than `DefaultRegistry` here? It seems like we should be able to get that information out of the manifest if we need it

Reply:
Doesn't seem like it and I agree. I'm not sure when we would ever want to start resolving from somewhere other other than the default registry anyway.